src/xdocs/google_style.xml

```diff
@@ -616,7 +616,7 @@
                 </td>
                 <td>
                   <a href="styleguides/google-java-style-20180523/javaguide.html#s4.1.1-braces-always-used">
-                    4.1.1 Braces are used where optional</a>
```

Comment:
I haven't updated the cached page reference because my this PR is still not merged yet: https://github.com/checkstyle/checkstyle/pull/14900